app/views/layouts/application.html.haml

```diff
@@ -10,10 +10,14 @@
   %body
     .container
       %ul.nav
+        %li.titlehead MovieCannon
```

Comment:
I'd recommend checking out the [navbar brand](https://getbootstrap.com/docs/4.3/components/navbar/#brand) from bootstrap for a more bootstrappy way to implement this!

Reply:
Awesome, using a navbar tool now. Thanks